Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -22,7 +22,11 @@
 use Symfony\Component\HttpKernel\Log\LoggerInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
-class AdminExtractor implements ExtractorInterface, TranslatorInterface, SecurityHandlerInterface, LabelTranslatorStrategyInterface
+final class AdminExtractor implements
+    ExtractorInterface,
```

Comment:
> Keep one line signature, please.

It's already 131 chars before the change, so no.
I don't see why there should be exceptions to the 120 rule (which is already pretty long, considering some coding standards like PEP-8 use less than 80 chars).

> BTW, the goal of this PR is to close this class by making it final, let keep the final diff minimal.

You should review PRs commit by commit, by using the "Showing changes from all commits" select box above.